common/test/acceptance/pages/lms/edxnotes.py

```diff
@@ -34,9 +34,9 @@ def _get_element_text(self, selector):
             return None
 
 
-class EdxNotesPageGroup(NoteChild):
+class EdxNotesChapterGroup(NoteChild):
     """
-    Helper class that works with note groups on Note page of the course.
+    Helper class that works with chapter (section) grouping of notes in the Course Structure view on the Note page.
```

Comment:
I believe it is to do with doc generation, so I guess it doesn't matter for acceptance tests. I've seen many PRs where folks have been commenting on this, so I thought I'd mention it. I know that the DRF APIs had to be kept narrower.